### ISSUE TITLE
Bitbucket support

### ISSUE THREAD

**Issue by brianmay** [pyupio/pyup#218]
Bitbucket support would be nice :-)

Alternatively, if there is anything similar that works for bitbucket...

**Comment by Neuro17** [comment]
Are there any news on this feature? :+1:

**Comment by jayfk** [comment]
No. And I'm afraid there are currently no plans to add bitbucket support. This might change in the future, so I'm leaving this issue open.

**Comment by iwan933** [comment]
Seems there is some wip. #349
